### DISCUSSION TITLE
Executing git commands on a build server

### DISCUSSION THREAD

**Discussion by camerongermein** [czproject/git-php#94] [General]
Do you think it's possible to execute all the git repo commands on an external build server, perhaps through SSH? Would it require building an all new Runner, or could we in theory just go: 

`$git = new CzProject\GitPhp\Git;`
`$repo = $git->open('ssh://user@host:/path/to/repo');`

**Comment by janpecha** [comment]
Hello, I didn't test it but I think you will need new Runner. Current Runner only executes Git commands on local machine.
